omod/pom.xml

```diff
@@ -85,6 +85,10 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>fhir2-omod</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>emrapi-api</artifactId>
```

Comment:
This needs to be added to the required_modules in `config.xml`.

Reply:
I will add this.